frontend/public/components/service-account.jsx

```diff
@@ -61,8 +65,21 @@ const Details = ({ obj: serviceaccount }) => {
     metadata: { namespace },
     secrets,
   } = serviceaccount;
-  const filters = { selector: { field: 'metadata.name', values: new Set(_.map(secrets, 'name')) } };
+  const serviceAcctSecrets = [...new Set(secrets?.map((s) => s.name))];
   const { t } = useTranslation();
+  const [resources, loaded, loadError] = useK8sWatchResource({
+    groupVersionKind: {
+      group: SecretModel.apiGroup,
+      kind: SecretModel.kind,
+      version: SecretModel.apiVersion,
+    },
+    isList: true,
+    namespaced: true,
+    namespace,
+  });
+  const [data, filteredData, onFilterChange] = useListPageFilter(resources, [], {
+    'secrets-service-account': { selected: serviceAcctSecrets },
```

Comment:
I think the `selected ` and `all` in staticFilter` type doesn't add any value. Wondering if it is a bug. 
https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts#L460
https://github.com/openshift/console/blob/master/frontend/public/components/factory/ListPage/filter-hook.ts#L21